code/datums/status_effects/neutral.dm

```diff
@@ -293,6 +301,23 @@
 /datum/status_effect/high_fiving/proc/dropped_slap(obj/item/source)
 	slap_item = null
 
+//this effect gives the user an alert they can use to surrender quickly
+/datum/status_effect/surrender
+	id = "surrender"
+	duration = -1
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/surrender
+
+/atom/movable/screen/alert/status_effect/surrender
+	name = "Surrender"
+	desc = "Looks like your in trouble now, bud. \ Click here to surrender (Warning: you will be paralyzed for 20 seconds)"
```

Comment:
```suggestion
	desc = "Looks like you're in trouble now, bud. \ Click here to surrender (Warning: you will be paralyzed)."
```

I have removed the timer as I don't think it's important and don't want two places to have to update.

Also, what's the backslash for? This looks like it's just going to create a second space.

Reply:
yeah sorry, I've been off of beyond for a while now and I instinctually went to make a new line after the sentence. It's unimportant and Ill remove it.